src/AssemblySharedInfoGenerator/AssemblySharedInfo.tt

```diff
@@ -9,7 +9,7 @@ using System.Runtime.InteropServices;
 // associated with an assembly.
 [assembly: AssemblyCompany("Autodesk, Inc")]
 [assembly: AssemblyProduct("Dynamo")]
-[assembly: AssemblyCopyright("Copyright © Autodesk, Inc 2014")]
+[assembly: AssemblyCopyright("Copyright © Autodesk, Inc 2015")]
 [assembly: AssemblyTrademark("")]
```

Comment:
No, it's version Copyright on Dynamo dlls, it doesn't matter if it's for Revit2014 or Revit2015. It's being built in year 2015, so the copyright should represent current year always.